src/collections/_documentation/data-management/event-grouping/grouping-enhancements.md

```diff
@@ -79,23 +79,23 @@ There are some general recommendations we have to greatly improve the out of the
 
 **Mark in-app Frames**
 
-To proactively improve your experience, help Sentry determine which frames in your stack trace are "in-app" (part of your own application) and which ones are not. The default rules are defined by the SDK, but in many cases, this can be improved on the server as well. In particular, for languages where server-side processing is necessary (For example, Native C, C++, or JavaScript.) it's better to override this on the server.
+To proactively improve your experience, help Sentry determine which frames in your stack trace are "in-app" (part of your own application) and which ones are not. The default rules are defined by the SDK, but in many cases, this can be improved on the server as well. In particular, for languages where server-side processing is necessary (For example, Native C, C++, or JavaScript) it's better to override this on the server.
```

Comment:
```suggestion
To proactively improve your experience, help Sentry determine which frames in your stack trace are "in-app" (part of your own application) and which ones are not. The default rules are defined by the SDK, but in many cases, this can be improved on the server as well. In particular, for languages where server-side processing is necessary (for example, Native C, C++, or JavaScript), it's better to override this on the server.
```